telink: tl721x: fix tercel pm mode.

haiwentelink opened the pull request:
- change power mode to DCDC_0P94_DCDC_1P8.
- set frequency to 80Mhz.
- set core to 0.9v ,and low down retention ldo.

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/d45ad1d5772528d8e3efc23fdcf8478cab7956f9 | https://github.com/telink-semi/hal_telink/commit/8e116bb74c344c043500425793d76c4d3ae21565 (develop) | [telink-semi/hal_telink@d45ad1d5..8e116bb7](https://github.com/telink-semi/hal_telink/compare/d45ad1d5772528d8e3efc23fdcf8478cab7956f9..8e116bb74c344c043500425793d76c4d3ae21565) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

s07641069 commented:
@haiwentelink Please create same PR into develop_new branch to keep changes synchronized

haiwentelink committed:
telink: tl721x: fix tercel pm mode.

- change power mode to DCDC_0P94_DCDC_1P8.
- set frequency to 80Mhz.
- set core to 0.9v ,and low down retention ldo.

Signed-off-by: Haiwen Xia <haiwen.xia@telink-semi.com>
